card/mod/05_standard/set/rstar/rules.rb

```diff
@@ -109,8 +117,10 @@ def default_open_rule_args args
 
   view :edit_rule, tags: :unknown_ok do |args|
     return 'not a rule' if !card.is_rule?
-    form_args = { url: path(action: :update, no_id: true),
-       html: {class: "card-form card-rule-form" } }
+    form_args = {
+       url: path(action: :update, no_id: true),
+       html: { class: "card-form card-rule-form" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.